lib/mwoffliner.lib.js

```diff
@@ -856,65 +931,82 @@ function execute(argv) {
 
   function saveArticles(dump, finished) {
     // these vars will store the list of js and css dependencies for the article we are downloading. they are populated in storeDependencies and used in setFooter
-    let jsConfigVars = '';
+    let jsConfiglets = "";
```

Comment:
You still need to fix this and review the file for any other search-replace instances that need to be undone.

Reply:
@darkenvy Do you still plan to go ahead with that merge request?

Reply:
Yes. I fixed all the commends that got replaced.

Reply:
But, I still see several instances of "jsConfigLets' in this file. Like the one above. Or did you forget to update the PR?

Reply:
@darkenvy Do not forget please to rebase, have done a few trivials fixes meanwhile.